apps/omg_child_chain_rpc/priv/swagger/operator_api_specs/transaction/request_bodies.yaml

```diff
@@ -4,6 +4,7 @@ TransactionSubmitBodySchema:
   content:
     application/json:
       schema:
+        title: 'TransactionSubmitBodySchema'
         type: object
```

Comment:
I mean this `type: object` is without apostrophes but it's a string as well - and also `description`